Applications/spire/source/charting/chart_view.cpp

```diff
@@ -12,9 +12,21 @@ using namespace Nexus;
 using namespace Spire;
 
 namespace {
-  QVariant chart_value_to_variant(ChartValue::Type type,
-      const ChartValue& value) {
-    return QVariant::fromValue(static_cast<Money>(value));
+  QVariant to_variant(ChartValue::Type type,
+      ChartValue value) {
+    if(type == ChartValue::Type::DURATION) {
+      return QVariant::fromValue(static_cast<time_duration>(value));
+    }
```

Comment:
```else if```

Reply:
Why is it else if here if it just returns? Optimization?

Reply:
No, no optimization happens. The principle is this... If I see this written as a series of ```else if```, I know as a reader that this giant if statement only ever executes one single branch. If I see a list of independent ```if``` statements that are not connected by an ```else/if```, I need to look into the body of every single ```if``` to see that each branch performs a ```return```.

As a code reviewer or someone reading this in the future, the ```else if``` is a very quick indicator to me that only one single branch has to be considered.